eodatasets3/stac.py

```diff
@@ -100,7 +100,7 @@ def _proj_fields(grid: Dict[str, GridDoc], grid_name: str = "default") -> Dict:
     """
     Add fields of the STAC Projection (proj) Extension to a STAC Item
     """
-    grid_doc = grid.get(grid_name)
+    grid_doc = grid.get(grid_name or "default")
```

Comment:
@jeremyh Can you please clarify what case this is handling, it's a confusing looking line of code now.

Reply:
I'll comment below, as there's maybe more investigation to do...